.gitignore

```diff
@@ -1,2 +1,5 @@
 dist
 .coverage
+
+# ignore goland files
+.idea
```

Comment:
Please add a newline at the end of files.

<img width="292" alt="image" src="https://github.com/aquaproj/aqua/assets/13323303/700b1fbc-d802-46ec-99ec-fce7b35267d8">